src/main/java/edu/ucsb/cs156/happiercows/helpers/CommonStatsCSVHelper.java

```diff
@@ -23,7 +23,7 @@ public class CommonStatsCSVHelper {
 
   private CommonStatsCSVHelper() {}
 
-  /**
+  /*
```

Comment:
No... this should be left the way it was.  This is a Javadoc comment, and the special `/** ` is needed to signal that.

See: https://www.baeldung.com/javadoc#javadoc-comments

See: 
```suggestion
  /**
```